src/core_plugins/timelion/server/handlers/__tests__/parse_sheet.js

```diff
@@ -0,0 +1,50 @@
+const chainRunnerFn = require('../chain_runner');
+const tlConfig = require('../../series_functions/__tests__/fixtures/tlConfig.js')();
+
+const expect = require('chai').expect;
+
+describe('timelion parse_sheet function', function () {
+  const chainRunner =  chainRunnerFn(tlConfig);
+
+  async function parse(data) {
+    const request = {
+      sheet: data,
+      time: {
+        from: 'now-15m',
+        to: 'now',
+        mode: 'quick',
+        interval: 'auto',
+        timezone: 'Europe/Berlin'
+      }
+    };
+    return await Promise.all(chainRunner.processRequest(request));
+  }
+
+  it('does not allow whitespace used between two expressions', async function () {
+    const data = ['.es() .es(404)'];
+    expect(function () {
+      parse(data);
+    }).to.throw();
```

Comment:
You shouldn't be calling up the interpreter to test this, rather get the properties on the AST. I explain the differences in the generated AST here: 

https://github.com/elastic/kibana/pull/10961#issuecomment-292534143

I should've been more specific in that comment. While interpreter tests would be nice the goal of this pull is only related to parsing and the tests should be testing the parser aspect.